cmd/leftovers/leftovers.go

```diff
@@ -32,6 +33,7 @@ func GetLeftovers(logger *app.Logger, o app.Options) (leftovers, error) {
 	case app.Azure:
 		l, err = azure.NewLeftovers(logger, o.AzureClientID, o.AzureClientSecret, o.AzureSubscriptionID, o.AzureTenantID)
 	case app.GCP:
+		fmt.Println("in gcp")
```

Comment:
Reminder to self and Gen: make sure this test line is gone before merging the PR